mcs_benchmark_data/loaders/rdf_file_loader.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         *,
         compress: bool = True,
         file_path: Optional[Path] = None,
-        format="json-ld",
+        format="turtle",
```

Comment:
ttl

Also, please don't check in the .ttl file, only the .ttl.bz2 produced by the loader.